app/controllers/v0/map_services_controller.rb

```diff
@@ -8,7 +8,8 @@ class MapServicesController < SignIn::ServiceAccountApplicationController
     # POST /v0/map_services/:application/token
     def token
       icn = @service_account_access_token.user_attributes['icn']
-      result = MAP::SecurityToken::Service.new.token(application: params[:application].to_sym, icn:)
+      cache = params[:cache] != 'false'
```

Comment:
I think we probably shouldn't cache the call from the controller, it seems to me like this is the one case where the caller would expect a new token with every new call

Reply:
That being said, I'd say we should set to to `true` for basically all the other MAP uses, so since the default is to make it `true` that should already accomplish that